{
  "cve_id": "CVE-2023-52948",
  "timestamp": "2025-01-14 16:58:45",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-10 10:40:16"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-11 20:55:47"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-12 23:10:06"
    }
  },
  "urls": [
    {
      "url": "https://www.synology.com/en-global/security/advisory/Synology_SA_24_11",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    }
  ]
}